packages/core/src/axis-chart.ts

```diff
@@ -38,14 +39,15 @@ export class AxisChart extends Chart {
 	}
 
 	protected getAxisChartComponents(graphFrameComponents: any[]) {
+		const options = this.model.getOptions();
 		const isZoomBarEnabled = Tools.getProperty(
-			this.model.getOptions(),
-			'zoomBar',
```

Comment:
you might have to try linting? it **should** be changing the double quotes to single quotes now ...